plutus-core/plutus-ir/test/PlutusIR/Transform/Inline/nameCapture.golden

```diff
@@ -1,5 +1,7 @@
 let
   !a_4 : integer = 10
+in
+let
```

Comment:
We could consider running `LetMerge` after the inliner now, since it splits up all non-recursive lets and doesn't put them back together again.

Reply:
Not a big deal, since we run `LetMerge` before the readable step, which is where it matters most, since everything else should handle split-up lets just the same. Might mean we're less likely to spot bugs relating to non-split lets :joy: 

I can do this nicely in the pass framework.

Reply:
Exactly, the single-lets will be merged later.